components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SQLQueries.java

```diff
@@ -327,4 +327,8 @@ public class SQLQueries {
             "WHERE EXISTS (SELECT SESSION_ID, TIME_CREATED FROM SESSIONS s WHERE ss.SESSION_ID = s.SESSION_ID AND " +
             "ss.TIME_CREATED = s.TIME_CREATED) AND OPERATION = ''STORE'' " +
             "AND SESSION_TYPE = ''AppAuthFrameworkSessionContextCache''{3} ORDER BY TIME_CREATED {4}";
+
+    public static final String SQL_GET_IDP_ID_BY_USER_ID = "SELECT IDP_ID FROM IDN_AUTH_USER WHERE USER_ID = ?";
+    public static final String SQL_GET_IDP_IDS_BY_USER_ID_LIST = "SELECT USER_ID, IDP_ID FROM IDN_AUTH_USER WHERE " +
```

Comment:
These two queries can be removed by returning idpIds by `getUserIdFromFederatedMapping`